examples/use-require.py

```diff
@@ -5,9 +5,11 @@
 def include(relative, filename):
     __file__ = str(Path(relative, filename).resolve())
     if (path.exists(__file__)):
-        fileHnd = open(__file__, "r")
-        exec(fileHnd.read())
-        return locals()
+        exports = {}
```

Comment:
Question: does it make sense to make CommonJS modules written in Python have Python-module semantics rather than CommonJS semantics? ... I don't think it does; one key observation with respect to a good module is that how you use a given module should not change regardless of what source language it was written in. `.node` modules, for example, are .so files that know how to mutate a pointer to an exports object; we don't just grab whatever symbols are visible in the solib.

I think your scoping variable should probably be called something other than exports, and then we should return `scope.exports` here?  That gets the good machinery and the CommonJS semantics when loading .py files with require()

I guess you're probably thinking about loading non-CommonJS modules..... I am hoping we can import Python modules with ESM import, but I haven't done the work there yet to know for sure if it's possible. I think the semantics match better that way, though?

Reply:
Discussed in slack and I think we came to a resolution.